.github/workflows/translations-upload.yml

```
@@ -32,3 +32,6 @@ jobs:
         upload_translations: false
 
         download_translations: false
+              - name: crowdin-action
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)